windows_containers/windows-containers-release-notes-3-x.adoc

```diff
@@ -17,6 +17,39 @@ The release notes for Red Hat OpenShift for Windows Containers tracks the develo
 
 You must have a subscription to receive support for the Red Hat WMCO. Deploying Windows container workloads in production clusters is not supported without a subscription. If you do not have a subscription, you can use the community WMCO, a distribution that lacks official support. Request support through the link:http://access.redhat.com/[Red Hat Customer Portal].
 
+[id="wmco-3-1-0"]
+== Release notes for Red Hat Windows Machine Config Operator 3.1.0
+
+Issued: 2021-09-21
+
+The WMCO 3.1.0 is now available with bug fixes and a new feature. The components of the WMCO were released in link:https://access.redhat.com/errata/RHBA-2021:3215[RHBA-2021:3215].
```

Comment:
Would it make sense to spell out WMCO on first reference here?

Reply:
The intro of the release notes gives the full name, so I think the shorthand reference here should be safe.